dapp/src/theme/Progress.ts

```diff
@@ -8,48 +8,54 @@ import { keyframes } from "@chakra-ui/react"
 
 const multiStyleConfig = createMultiStyleConfigHelpers(parts.keys)
 
-const filledStyle = defineStyle((props) => {
-  const { colorScheme: color, theme, isIndeterminate, hasStripe } = props
+const trackStyle = defineStyle((props) => {
+  const { isIndeterminate, hasStripe } = props
 
-  const bgColor = `${color}.400`
-  const stripeColor = "opacity.white.1"
+  const bgColor = "#3A3328"
```

Comment:
Let's define this color. If it's only used in the `Progress` component we can declare a const value but outside of the function - for example at the beginning of this file.